apps/sensors_test/src/main.c

```diff
@@ -413,10 +410,10 @@ config_sensor(void)
 
     dev = (struct os_dev *) os_dev_open("light0", OS_TIMEOUT_NEVER, NULL);
     assert(dev != NULL);
-    rc = tsl2561_init(dev, NULL);
-    if (rc) {
-        os_dev_close(dev);
-        goto err;
+
+    if (dev->od_flags & OS_DEV_F_STATUS_READY) {
+        rc= SYS_ENODEV;
+        console_printf("config_sensor %s init failed\n", dev->od_name);
```

Comment:
Maybe it is worth to mention that in commit message then.

BTW. There are two patches with same title. Would it be possible to change title for them?  That would really helps later when you search something in the git tree e.g. using "git log --oneline"  
What we try to do is to keep MYNEWT Jira as a prefix and then high level Title. If we do work without JIRA ticket we put submodule name as a prefix. In commit message we usually put what we change and more important why we change it.